src/main/java/Dross.java

```diff
@@ -0,0 +1,87 @@
+import java.util.Scanner;
+
+public class Dross {
+    private static List drossTaskList = new List();
+    private static Scanner in = new Scanner(System.in);
+
+    //Method to print a line of _ characters
+    public static void printLine() {
+        System.out.println("_".repeat(55));
+    }
+
+    //Method to read in a line of text from standard input
+    public static String readLine() {
+        return in.nextLine().trim();
+    }
+
+    //Method to exit Dross bot
+    public static void goodbye() {
+        printLine();
+        System.out.println("Bye. Hope to see you again soon!");
+        printLine();
+    }
+
+    //Method to list all tasks
+    public static void listAllTasks() {
+        printLine();
+        drossTaskList.printAllTasks();
+        printLine();
+    }
+
+    //Method to toggle tasks as marked or unmarked
+    public static void toggleMark(String instruction) {
+        String[] tokens = instruction.split(" ");
```

Comment:
Good eye for details! Plural form was used for the array names consistently throughout.